client/js/components/procedure/admin/InstitutionTagManagement/InstitutionList.vue

```diff
@@ -232,6 +232,7 @@ export default {
   mixins: [tableScrollbarMixin],
 
   props: {
+    // Array when empty, object when not empty
```

Comment:
:-/

Reply:
can't we send always an object?

Reply:
As a matter of fact, we can. I thought that since it is initially set to `this.initialFilter`, we couldn't, because it has type `[Object, Array]`, and normally that's because of how we get it from the BE via twig. But as I realized now, `initialFilter` is never passed to the component, so it can be removed, and then `appliedFilterQuery` can just be an object. Thanks for asking!

Reply:
Done in df3e25905f6f5aa131743aa791ba8a52a6e61cca